src/Equinor.ProCoSys.IPO.Infrastructure/EntityConfigurations/InvitationConfiguration.cs

```diff
@@ -38,14 +38,12 @@ public void Configure(EntityTypeBuilder<Invitation> builder)
             builder
                 .HasMany(x => x.McPkgs)
                 .WithOne()
-                .IsRequired()
-                .OnDelete(DeleteBehavior.NoAction);
+                .IsRequired();
 
             builder
                 .HasMany(x => x.CommPkgs)
                 .WithOne()
-                .IsRequired()
-                .OnDelete(DeleteBehavior.NoAction);
```

Comment:
What does this remove do?

Reply:
If we don't specify .OnDelete, the default DeleteBehavior will be Cascade delete (which you see in Migrations/20210325125330_ScopeDeleteChange.cs)
In this PR you'll also see that the handler for edit ipo do not excplicit delete any CommPkgs/McPkgs longer (ref deleted methods in InvitationRepository). We just remove the mcpkg/commpkg from the list of them in invitation, and the cascade delete setting performs the actual delete of such mcpkg/commpkg from corresponding tables in DB.

U see?